package.json

```diff
@@ -36,7 +36,7 @@
     "@adobe/magento-storefront-event-collector": "^1.8.0",
     "@adobe/magento-storefront-events-sdk": "^1.8.0",
     "@dropins/storefront-account": "~1.0.4",
-    "@dropins/storefront-auth": "~1.0.6",
+    "@dropins/storefront-auth": "~2.0.0",
```

Comment:
Was there no change to integration code with this release? Is the use of `is_subscribed` managed entirely within auth containers/components?